perlmod/Fink/Services.pm

```diff
@@ -594,6 +599,32 @@ EOSCRIPT
 		@wrap = map "$_=$ENV{$_}", sort keys %ENV;
 		push @wrap, "__CFPREFERENCES_AVOID_DAEMON=1";
 		unshift @wrap, 'env' if @wrap;
+		my $runtime_request = Fink::Config::get_option("build_in_sandbox");
```

Comment:
Or do you just mean that I need to bump...

our $conf_file_compat_version  = 3;

in perlmod/fink/Configure.pm 

to 

our $conf_file_compat_version  = 4;

due to the new NoSandbox field being supported in Info files?